documentation/release_3.1.htm

```diff
@@ -285,6 +292,7 @@ <h3>Backward incompatibities</h3>
     now returns an extra segment pair if the maximum ring difference does not coincide with a &quot;full&quot; segment.
     This change was to accomodate how GE uses even span.
 </li>
+<code>CListModeData</code> now need to implement a <code> ProjDataInfo</code> instead of <code>Scanner</code>. This will probably break all related classes. 
```

Comment:
"Classes derived from CListModeData ..."
also not clear what is meant with `need to implement`